layouts/community/ergodox/zweihander-macos/rules.mk

```diff
@@ -1,3 +1,9 @@
 # Don’t do a tricolor flashing-light wave when the computer is shut down in Windows,
 # or when the machine is sleeping in macOS
 SLEEP_LED_ENABLE = no
+
+# Don’t pretend to be a mouse according to <https://docs.qmk.fm/#/feature_pointing_device?id=troubleshooting>
+POINTING_DEVICE_ENABLE = no
+
+# Don’t pretend to be a mouse according to <https://github.com/qmk/qmk_firmware/issues/510#issuecomment-232067620> (actually worked)
```

Comment:
No links in makefiles, please. 

```suggestion
# Don’t pretend to be a mouse
POINTING_DEVICE_ENABLE = no
```

Reply:
I tried having just `MOUSE_ENABLE = no` in the `rules.mk` for this and I still got the cursor in FreeBSD. This next update seems to pare things down to the bare essentials. Oddly enough, running `diff -u` between the generated .hex files of both my original commit (https://github.com/qmk/qmk_firmware/pull/20997/commits/f35a335c53e356f9ad680b60966aa9b235918937) and the current one (https://github.com/qmk/qmk_firmware/pull/20997/commits/092fa0cab941d944d09499d5360acb5a9618b7c4) says there's no difference.